yarn.lock

```diff
@@ -19048,6 +18964,39 @@ __metadata:
   languageName: node
   linkType: hard
 
+"depcheck@npm:^1.4.7":
+  version: 1.4.7
+  resolution: "depcheck@npm:1.4.7"
+  dependencies:
+    "@babel/parser": "npm:^7.23.0"
+    "@babel/traverse": "npm:^7.23.2"
+    "@vue/compiler-sfc": "npm:^3.3.4"
```

Comment:
Sadly, It has a quite a lot of dependencies :(

There is probably nothing we can do about it, you can ignore my comment here.